apps/public-reference/pages/applications/contact/alternate-contact-type.tsx

```diff
@@ -20,11 +20,12 @@ export default () => {
   const { register, handleSubmit, errors, watch } = useForm<Record<string, any>>()
   const onSubmit = (data) => {
     application.alternateContact.type = data.type
+    conductor.completeStep(1)
     conductor.sync()
     if (data.type == "noContact") {
-      Router.push("/applications/household/live-alone").then(() => window.scrollTo(0, 0))
+      conductor.routeTo("/applications/household/live-alone")
     } else {
-      Router.push("/applications/contact/alternate-contact-name").then(() => window.scrollTo(0, 0))
+      conductor.routeTo("/applications/contact/alternate-contact-name")
```

Comment:
Why not routeToNextOrReturnUrl here?

Reply:
I think it's because there's not a return link at the bottom of this page…I started to add one but realized if somebody changes the value here, they'd have to go to the next page to complete the related information.